digiroad2-geo/src/main/scala/fi/liikennevirasto/digiroad2/TrafficSign.scala

```diff
@@ -30,6 +31,9 @@ sealed trait TrafficSignType {
 
   def group: TrafficSignTypeGroup
 
+  //This is only used to put CycleAndWalkwaySigns group at the moment
+  def additionalGroup: Option[TrafficSignTypeGroup] = None
+
```

Comment:
If we start to having more signs with "mixed" groups maybe we should rethink the `def group: TrafficSignTypeGroup` to `def group: Seq[TrafficSignTypeGroup]` or something like that.